recipes_source/distributed_device_mesh.rst

```diff
@@ -148,6 +148,26 @@ Then, run the following `torch elastic/torchrun <https://pytorch.org/docs/stable
 
     torchrun --nproc_per_node=8 hsdp.py
 
+How to use DeviceMesh for your custom parallel solutions
+--------------------------------------------------------
+When working with large scale training, one can have a more complicated custom parallel training composition. For example, one may need to slice out submeshes for different parallelism solutions.
```

Comment:
```suggestion
When working with large scale training, you might have more complex custom parallel training composition. For example, you may need to slice out sub-meshes for different parallelism solutions.
```

Reply:
If this is for 2.4, can you please label with the 2.4 label.

Reply:
@svekars Not sure why but it seems I can't add reviewer and labels. I was gonna add you and one team member @wanchaol as the reviewer.